async_pi_estimate/node-addon-api/pi_est.h

```diff
@@ -0,0 +1,14 @@
+/*********************************************************************
+ * NAN - Native Abstractions for Node.js
+ *
+ * Copyright (c) 2018 NAN contributors
+ *
+ * MIT License <https://github.com/nodejs/nan/blob/master/LICENSE.md>
+ ********************************************************************/
```

Comment:
I don't see this copyright in any of the files being added under nan. I'd expect it to be in both or neither.

Reply:
Removed copyright